proposals/collection-literals.md

```diff
@@ -297,14 +335,25 @@ var y = First(new int[0], [1, 2, 3]);    // ok: int[]
 static T First<T>(T x, T y) => x;
 ```
 
-To allow conversions to other collection types, type inference may need to infer exact, lower-bound, and upper-bound inferences from the **expression `U`** to a type `V` when an expression is given, and *fixing* §11.6.3.12 updates the sets of *candidate types* based on implicit conversion from the **expression** bound to the each *candidate type*.
+To allow conversions to other collection types, type inference may need to infer exact, lower-bound, and upper-bound inferences from the **expression `U`** (rather than the type of `U`) to a type `V` when an expression is given, and *fixing* §11.6.3.12 updates the sets of *candidate types* based on implicit conversion from the **expression** bound to the each *candidate type*.
 
 _Should nested cases infer collection types successfully?_
 ```c#
 var x = new[] { [ulong.MaxValue], [1, 2, 3] };  // ok: List<ulong>[]
 var y = First([[ulong.MaxValue]], [[1, 2, 3]]); // ok: List<List<ulong>>
 ```
 
+## Overload resolution
+_Include betterness order - perhaps the following, from best to worst: spans; arrays and constructed types; interface types._
+
+_Which overload should we prefer in the following?_
+```c#
+F([1:2]); // ambiguous?
```

Comment:
I don't see why this would be ambiguous if it's not ambiguous today. `IDictionary<K, V>` is the more specific type. What would be more interesting to talk about is something like `IEnumerable<KVP<K, V>>` vs `IDictionary` (no generic), as I expect that would be ambiguous today.

Reply:
Thanks. Changed to `IDictionary`.